controllers/etcdcopybackupstask/reconciler.go

```diff
@@ -140,8 +134,8 @@ func (r *Reconciler) doReconcile(ctx context.Context, task *druidv1alpha1.EtcdCo
 		return status, nil
```

Comment:
Are the jobs getting removed automatically? I think there was another issue where it was found that the TTL was not set on the jobs that enables auto removal. So imagine a case where the user creates a EtcdCopyBackupsTask. It gets completed and he removes the task but the job is still there (check this). Now create another task with the same name. Since the job with name `<task-name>-worker` already exists then no new Job will be created. Can you check if this is the behavior?

Reply:
I am adding `EtcdCopyBackupsTask ` as ownerreference to job `<task-name>-worker` [here](https://github.com/gardener/etcd-druid/pull/553/files#diff-50d8f23e4c9d06e06ce9338c5d105b870667137a087aa14dc4f0a257da4c7730R372-R374).
This ensures that once the `EtcdCopyBackupsTask` task is deleted, its dependents will also be deleted.